aspnetcore/tutorials/razor-pages/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Create a Razor Pages web app with ASP.NET Core
+author: rick-anderson
+description: 
+keywords: ASP.NET Core, Razor Pages, scaffolding, Entity Framework Core, EF, EF Core, database
```

Comment:
According to Gauntlet, the spaces after the commas in the keywords metadata should be removed.